### PR TITLE
Add summary to Alertmanager rules where missing

### PR DESCRIPTION
**Pull request by alexanderdavidsen**
Added summary to Alertmanager rules where missing. The summary is based off the description label, but contains less information - ideal for headers when sending notifications to Slack or HipChat.

### CODE REVIEW

**Comment by brancz** [comment]
Could you add these to the alert definitions in `contrib/kube-prometheus/assets/prometheus/rules` and the run `make generate` from the `contrib/kube-prometheus/` directory? Otherwise lgtm!

**Comment by gianrubio** [comment]
To update the helm charts you should run [this script](https://github.com/coreos/prometheus-operator/blob/master/helm/hack/sync_kube_prometheus.py)  and follow [this steps](https://github.com/coreos/prometheus-operator/tree/master/helm#how-to-contribue)

File: helm/grafana/templates/grafana-dashboards.yaml [hunk 1]
```diff
@@ -1,7486 +1,9 @@
 {{ define "grafana-dashboards.yaml.tpl" }}
-deployment-dashboard.json: |
```
Comment (brancz) [inline helm/grafana/templates/grafana-dashboards.yaml]: I think this got deleted by accident.

Comment (alexanderdavidsen) [reply helm/grafana/templates/grafana-dashboards.yaml]: I guess one of the scripts removed this line. I executed "make generate" in contrib/kube-prometheus and ran helm/hack/sync_kube_prometheus.yaml. 

Should "make generate" create grafana-dashboard.json?

Comment (brancz) [reply helm/grafana/templates/grafana-dashboards.yaml]: I'm assuming something there went wrong, it's probably best to just revert the changes in this file, but ultimately we should fix the scripts. (cc @gianrubio )

Comment (alexanderdavidsen) [reply helm/grafana/templates/grafana-dashboards.yaml]: I have reverted grafana-dashboards.yaml to it's previous state and pushed the new commit. Should be building now.

**Comment by brancz** [comment]
lgtm on green

**Pull request merged**